pkgs/top-level/all-packages.nix

```
@@ -4644,6 +4644,8 @@ in
 
   ola = callPackage ../applications/misc/ola { };
 
+  olive-editor = callPackage ../applications/video/olive-editor { };
```

Qt applications need to use a special `callPackage` ` libsForQt5.callPackage`.

> nixpkgs manual
> https://nixos.org/nixpkgs/manual/#sec-language-qt

---

It's also a good idea to inherit the attributes for darwin support here like
```suggestion
  olive-editor = callPackage ../applications/video/olive-editor {
    inherit (darwin.apple_sdk.frameworks) CoreFoundation;
  };
```

---

Makes sense.